src/utils/downloadFileByLink.js

```diff
@@ -0,0 +1,12 @@
+export const downloadFileByLink = (fileName, downloadLink) => {
+  if (!fileName || !downloadLink) return;
+
+  const elem = window.document.createElement('a');
+
+  elem.href = downloadLink;
+  elem.download = fileName;
```

Comment:
I suggest adding elem.style.visibility = 'hidden'; to be sure that we will note have any visible effect